lxd/storage/backend_lxd.go

```diff
@@ -2975,6 +3046,14 @@ func (b *lxdBackend) CreateInstanceSnapshot(inst instance.Instance, src instance
 		return err
 	}
 
+	parentUUID := srcDBVol.Config["volatile.uuid"]
+	if parentUUID == "" {
+		return fmt.Errorf(`Failed loading "volatile.uuid" from instance volume %q`, src.Name())
```

Comment:
"loading" suggests a retrieval from an external system (like a DB or API) IMHO, how about using "Parent instance volume is missing the required "volatile.uuid" setting..."?

Reply:
Fine for me, like it.